src/components/ForecastDetails.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import '../styles/ForecastDetails.css';
```

Comment:
This CSS file is empty. Having empty files committed to a project is always a bad idea, this should be removed until there is some content to define in there, or at the least using some placeholder values just to check the import is working